frontend/src/components/Map/index.js

```diff
@@ -329,6 +332,62 @@ export default function Map(props, { draggable = true }) {
     );
   };
 
+  const addCircle = (status, content) => {
+    const color = status.color;
+    const word = status.name;
+    content +=
+      '<div style="position:relative;width: 8px; height: 8px;line-height:8px;font-size:8px;' +
+      "margin-right: 10px;top:5px;float: left;border-radius: 50%;background:";
+    content = content + color + ';"></div>';
+    content =
+      content +
+      '<p style="position:relative;top:5px;right:5px;float:left;' +
+      "color:" +
+      color +
+      ';line-height:8px;font-size:8px;">' +
+      word.toUpperCase() +
+      "</p>";
+    return content;
+  };
+  const popUpContent = (userStory, content) => {
+    if (userStory.age) content = content + " " + userStory.age + " years old";
+    content += userStory.myStory || userStory.age ? " user " : " User ";
+    if (userStory.profession !== "")
+      content =
+        content +
+        " working in the " +
+        userStory.profession.toLowerCase() +
+        " industry ";
+    content = content + "living near " + userStory.state;
+    var date = userStory.createdAt.substring(0, 10);
+    if (userStory.myStory) content = content + " on " + date;
+    content += ".</p>";
+    content += '<div style="line-height:8px;" class="row">';
+    content = addCircle(statusMapping[userStory.sick], content);
+    content = addCircle(statusMapping[userStory.tested], content);
+    content += "</div>";
+    console.log(content);
+    return content;
+  };
+
+  const setHover = (marker, content, map) => {
+    var popup = new mapboxgl.Popup({
+      closeButton: false,
+      closeOnClick: false,
+      offset: 25,
+    });
+    popup.setHTML(content);
+    const element = marker.getElement();
+    element.id = "marker";
+    // hover event listener
+    element.addEventListener("mouseenter", () => popup.addTo(map));
+    element.addEventListener("mouseleave", () => popup.remove());
+    // add popup to marker
+    marker.setPopup(popup);
+    // add marker to map
+    marker.addTo(map);
```

Comment:
Put these 2 lines outside of this function. It would make more sense as the 2 lines are not related to set hover-over popup.